.travis.yml

```diff
@@ -22,9 +22,7 @@ services:
     - elasticsearch
 
 before_script:
-    - git clone https://${GITHUB_TOKEN}:x-oauth-basic@github.com/juttle/juttle.git
-    - pushd juttle ; npm install ; npm link ; popd
+    - npm install juttle@0.1.x
 
 script:
-    - export NODE_PATH="$(dirname `which node`)/../lib/node_modules"
     - mocha
```

Comment:
In juttle-sql-adapter-common and juttle-graphite-adapter we've started putting the section on scripts into the package.json like so:

https://github.com/juttle/juttle-sql-adapter-common/blob/master/package.json#L10

and then in the travis file you can lint and run the tests through `npm`:

https://github.com/juttle/juttle-sql-adapter-common/blob/master/.travis.yml#L26